### DISCUSSION TITLE
Query: GUI redesign

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1210] [Q&A] [answered]
@midwan ~ Kudos for all the improvements here ...

...if you customize GUI colours, you can end up looking like this...

![example](https://imgur.com/PZfrIlQ.png)

...the 'vacant' area bottom left ...previously due to the fact the buttons filled that side panel, you'd never see this ; the button background colour, pretty much coloured the side panel ... now the GUI background colour can be seen ....

Not that I'm complaining at all, it just looks a bit 'goofy' =) 

I'm just wondering is this is an unexpected consequence of changing GUI colours with the new redesign? ...ie; there is no problem with default colours...

...or...do you have future plans for this area? That is, it's obvious it allows expansion for additional buttons, but right now it's an 'unused' GUI area...ie; it does nothing =) 

I'm not sure if there's a 'cosmetic' fix... the probably looks like finding that area, defining it as a button (with no fill/function), so it inherits the same button background colour, or such and similar...

...but right now, even having the Amiberry logo propped there does something (advertising =) ... and that could be a 'Quick Launch' button, that users could configure (amiberry.conf) to ie; start/run their favourite Workbench setup in emulation.... but like, I'm just trying to think of 'anything/something' to put in that GUI void... B^)

Comments?

TIA

**Comment by midwan** [comment]
It looks fine with the default colors, as the background of the non-selected item is the same color as the background of the area there. :)

**Reply by midwan** [reply, marked as answer]
@giantclambake fixed with 8f9cb16e26ba8621bfea6bebef3e0225595f0cbd

**Comment by giantclambake** [comment]
Thanks, looks better  ;)

![ex](https://imgur.com/mWtUOJS.png)
